src/openrct2/rct2/S6Exporter.cpp

```diff
@@ -1173,7 +1173,7 @@ void S6Exporter::ExportSpritePeep(RCT2SpritePeep* dst, const Peep* src)
     {
         dst->ride_types_been_on[i] = src->RideTypesBeenOn[i];
     }
-    dst->item_extra_flags = src->ItemExtraFlags;
+    dst->item_extra_flags = src->GetItemFlags(true);
```

Comment:
will need reworking when you change the functions as suggested.

Reply:
now this function has two parameters, which make it to work like directly accessed flags. On the next PR we could without much work make the function to return uint64_t with one simple change to funtion signature.